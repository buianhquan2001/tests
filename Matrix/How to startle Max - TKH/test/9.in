30
1748
1884
1660
1735
1916
1427
1712
1690
1012
1302
1744
1924
1639
1748
1389
1407
1258
1315
1270
1982
1155
1104
1381
1081
1603
1013
1205
1569
1769
1250
