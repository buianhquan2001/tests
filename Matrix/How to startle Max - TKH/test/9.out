726389763
593105869
199790517
991248350
734247522
364780689
325878177
339483698
683718464
873330960
93175058
65817473
121168533
726389763
393419388
757666747
210974181
794680339
909809121
385004325
465967079
762419651
390697384
363498772
427399457
123542166
218921398
374163730
701140010
635993383
